roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -225,6 +225,7 @@ kiali_defaults:
     port: 20001
     web_fqdn: ""
     web_history_mode: ""
+    web_port: 20001
```

Comment:
```suggestion
    web_port: ""
```

Reply:
Note - since this appears to be a numeric value - do we want the default to be `0` or empty string `""`? I don't know what the backend will want.
@israel-hdez @cfcosta